src/users/dto/update-user.dto.ts

```diff
@@ -0,0 +1,6 @@
+export class UpdateUserDto {
+    name: string;
+    email: string;
+    account_status: boolean;
+    password: string;
+}
```

Comment:
When updating, all fields could potentially be undefined/null, since a User only updates one thing at a time in most cases.
The Account Status is determined by the system similar to what I mentioned above :arrow_up: 

:wrench: Therefore we probably need to make all fields optional:
```suggestion
export class UpdateUserDto {
    name?: string;
    email?: string;
    password?: string;
}
```